Automattic/jetpack
# Change remaining shares phrasing to be more clear

This small changes updates some strings to make the message more clear for users about their remaining shares.

### Other information:


## Jetpack product discussion



1203640801049718-as-1203656944717119

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






* Check all occurrences from Google doc in the Asana task
* The pricing page change is not included as that was already changed in a previous change.
